### PR TITLE
2.0.1

### PR DESCRIPTION
**Pull request by grawk**
changelog and package.json changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v2.0.1
+
+-- fix #176
```
Comment (aredridel) [inline CHANGELOG.md]: That could be more descriptive.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
+## v2.0.0
+
+-- Add new makara/internationalization support
```
Comment (aredridel) [inline CHANGELOG.md]: - And completely refactor
- And move config values from package.json to .yo-rc.json
- and fix some bugs if I remember right
- modernize and use new yeoman infrastructure

Comment (grawk) [inline CHANGELOG.md]: Thanks. Will add that.

Comment (grawk) [inline CHANGELOG.md]: er. not verbatim.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## v2.0.1
+
+-- fixes #176 by surrounding comma-separated task list with `[]` in the Gruntfile `build` task definition
+
+## v2.0.0
+
+-- new makara/internationalization support
```
Comment (aredridel) [inline CHANGELOG.md]: Let's use markdown and `* item` rather than the badly-rendering `-- item`. Below in the file too.

**Comment by grawk** [comment]
bump

**Pull request merged**